routes/api/user/index.js

```diff
@@ -26,35 +26,24 @@ router.get('/', authorization.needed('admin'), (req, res, next) => {
       .limit(limit),
     User.count()
   ])
-  .then(([data, count]) => {
-    const users = data.map((user) => {
-      const {id, displayName, created_at} = user;
-      return {
-        id,
-        displayName,
-        created_at,
-        profiles: user.toObject().profiles,
-        roles: user.toObject().roles
-      };
-    });
+  .then(([result, count]) => {
 
     res.json({
-      result: users,
+      result,
       limit: Number(limit),
       count,
       page: Number(page),
-      totalPages: Math.ceil(count / limit)
+      totalPages: Math.ceil(count / (limit === 0 ? 1 : limit))
```

Comment:
You can get `NaN` or `Infinity` if you enter `0` for different things here, but I don't care if you don't since it doesn't crash the app.